hack/run_test

```diff
@@ -498,6 +498,31 @@ $volume_options
   echo "  Success!"
 }
 
+run_doc_check() {
+  local tmpdir=$(mktemp -d)
+  echo "  Testing documentation in the container image"
+  # Extract the help files from the container
+  for f in help.md help.1 ; do
+    docker run --rm ${IMAGE_NAME} /bin/bash -c "cat /${f}" >${tmpdir}/${f}
+  done
+  # Check whether the files use the correct format
+  if ! file ${tmpdir}/help.1 | grep -q roff ; then
+    echo "ERROR: /help.1 is not in troff or groff format"
+    return 1
+  fi
```

Comment:
Makes sense to me, as soon as we start detaching parts of the tests to common part, this should be one of the first suspects. I've include that in the issue so we don't forget:
https://github.com/sclorg/container-common-scripts/issues/1